src/main/java/Duke.java

```diff
@@ -1,10 +1,100 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static String readPrompt(Scanner in){
+        return in.nextLine();
+    }
+/*
+    public static void echoPrompt(String input){
+        System.out.println(input);
+    }
+*/
+
+    public static void displayList(Task[] tasks, int taskIndex){
+        if (taskIndex==0){
+            System.out.println("\tYou haven't noted down anything yet.");
+            return;
+        }
+        System.out.println("\tHere are the tasks in your list:");
+        for (int i=0; i<taskIndex; i++){
+            System.out.println("\t" + (i+1) + "." + tasks[i].toString());
+        }
+    }
+
+    private static void addToList(Task[] tasks, String prompt, int taskIndex, String taskType) {
+        int cutOffPoint = 0;
+        switch(taskType) {
```

Comment:
Consider removing indentations for the case clauses. (It violates coding standard)

Reply:
Yes, you are right. Noted.